docs/en/Storage.md

```diff
@@ -1,132 +1,261 @@
-# Storage
+# Storage Types
 
-The platform has a range of different types of storage meant for a variety of
-use-cases, so this storage section applies whether you're experimenting,
-creating pipelines, or publishing.
+The platform provides two main types of storage:
 
-At the surface, there are two kinds of storage
+- Disk (also called Volumes on the Notebook Server creation screen)
+- Bucket ("Blob" or S3 storage, provided through MinIO)
 
-- Disks (also called Volumes)
-- Buckets (S3 or "Blob" storage)
+Depending on your use case, either may be most suitable:
+
+|   Type |                                                                                                         Simultaneous Users |                                                   Speed | Total size               |
+| -----: | -------------------------------------------------------------------------------------------------------------------------: | ------------------------------------------------------: | ------------------------ |
+|   Disk |                                       One compute instance at a time (one notebook server, step in kubeflow pipeline, ...) |                        Fastest (throughput and latency) | <=512GB total per drive  |
+| Bucket | Many compute instances at once (ex: multiple training processes on different notebook servers all accessing the same data) | Fast-ish (Fast download, modest upload, modest latency) | Infinite (within reason) |
+
+<!-- prettier-ignore -->
+??? info "If you're unsure which to choose, don't sweat it"
+    These are guidelines, not an exact science - pick what sounds best now and run with it.  The best choice for a complicated usage is non-obvious and often takes hands-on experience, so just trying something will help.  For most situations both options work well even if they're not perfect, and remember that data can always be copied later if you change your mind.
 
 ## Disks
 
-Disks are the familiar hard drive (or SSD) style file systems! You use them
-directly in Kubeflow when you add workspace and data volumes to your notebook
-server. They are automatically mounted at the directory you choose, and serve as
-a reliable way to preserve your data. Even if you delete your server later, you
-can still remount your disks to a new one – by default they are never destroyed.
+### Overview
+
+Disks are the familiar SSD style file systems! When creating your notebook
+server, you request them by adding Data Volumes to your notebook server. They
```

Comment:
I would add after "by adding Data Volumes to your notebook server" a little "(pictured below)"